packages/core/components/DraggableComponent/index.tsx

```diff
@@ -86,6 +89,22 @@ export const DraggableComponent = ({
     }
   }, []);
 
+  const loadedOverrides = useLoadedOverrides({
+    overrides: overrides,
+    plugins: plugins,
+  });
+
+  const DefaultActionBar = () => (
+    <ActionBar label={label}>
+      <DefaultOverride />
+    </ActionBar>
+  );
```

Comment:
Can this be pulled out to the top of the file, with `label` passed in as a prop?

Reply:
Done